spec/druuid_spec.rb

```diff
@@ -1,37 +1,42 @@
 require_relative '../druuid'
 
 describe Druuid do
-  describe '.gen' do
+  shared_examples "druuid generator" do |method|
```

Comment:
do we need to keep the `method` parameter? it seems like we're only ever passing in `:gen`